tests/MassTransit.Containers.Tests/Stop_Specs.cs

```diff
@@ -58,4 +58,66 @@ public Task Consume(ConsumeContext<PingMessage> context)
             }
         }
     }
+
+
+    [TestFixture]
+    public class Stop_Redelivery_Specs :
+        InMemoryTestFixture
+    {
+        [Test]
+        public async Task Should_respect_ConsumerStopTimeout()
+        {
+            var collection = new ServiceCollection();
```

Comment:
So, this should be `await using var provider` like the other tests to ensure a clean shutdown.